tests/functional/test_exception_hooks.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+
+from globus_sdk._testing import RegisteredResponse, load_response, load_response_set
+
+
+def test_session_required_policies(run_line):
+    """
+    ls on hits a 403 with session_required_policies set
+    confirms correct globus session update command in helptext
```

Comment:
This explanation isn't reading correctly the first and second time I read it. Would you see if there's a way to better explain this?

Reply:
Ah yeah, that needs another pass